utils/wasi-nn/install-openvino.sh

```diff
@@ -3,17 +3,18 @@
 # SPDX-FileCopyrightText: 2019-2022 Second State INC
 
 if [[ ! -v "${OPENVINO_VERSION}" ]]; then
-  OPENVINO_VERSION="2021.4.582"
+  OPENVINO_VERSION="2023.0.0"
 fi
 if [[ ! -v "${OPENVINO_YEAR}" ]]; then
-  OPENVINO_YEAR="2021"
+  OPENVINO_YEAR="2023"
 fi
 
 set -e
 echo "Installing OpenVINO with version ${OPENVINO_VERSION}"
-curl -sSL https://apt.repos.intel.com/openvino/$OPENVINO_YEAR/GPG-PUB-KEY-INTEL-OPENVINO-$OPENVINO_YEAR | gpg --dearmor > /usr/share/keyrings/GPG-PUB-KEY-INTEL-OPENVINO-$OPENVINO_YEAR.gpg
-echo "deb [signed-by=/usr/share/keyrings/GPG-PUB-KEY-INTEL-OPENVINO-$OPENVINO_YEAR.gpg] https://apt.repos.intel.com/openvino/$OPENVINO_YEAR all main" | tee /etc/apt/sources.list.d/intel-openvino-$OPENVINO_YEAR.list
+wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS.PUB
+echo "deb https://apt.repos.intel.com/openvino/2023 ubuntu20 main" | tee /etc/apt/sources.list.d/intel-openvino-2023.list
 apt update
-apt install -y intel-openvino-runtime-ubuntu20-$OPENVINO_VERSION
-source /opt/intel/openvino_2021/bin/setupvars.sh
+apt-get -y install openvino
+source /opt/intel/openvino_2023/bin/setupvars.sh
```

Comment:
Update: The error lies here , it says that no such file or directory.
I changed it from openvino_2021 to openvino_2023 but that was not the correct way to do it.